packages/buildessential.rb

```diff
@@ -100,8 +100,8 @@ class Buildessential < Package
 
   # LLVM
   # This can be pulled in on a per-package basis...
-  # depends_on 'llvm19_dev'
-  depends_on 'llvm19_lib' # This provides llvm-strip
+  # depends_on 'llvm20_dev'
+  depends_on 'llvm20_lib' # This provides llvm-strip
```

Comment:
Are we uninstalling older llvm versions when we upgrade?  The reason I ask is these packages are huge and it could fill up the hdd on low end Chromebooks quickly.

Reply:
We are not... but also, this is `buildessential`.

```
crew files llvm20_lib
llvm20_lib: LibLLVM and llvm-strip
/usr/local/bin/llvm-strip
/usr/local/lib64/libLLVM-20.so
/usr/local/lib64/libLLVM.so
/usr/local/lib64/libLLVM.so.20.1
Total found: 4
Disk usage: 424.14 MB
```

How would you suggest we autoremove?

The last time we had that it was broken: https://github.com/chromebrew/chromebrew/issues/6425

Reply:
Don't remove everything? LoL

Reply:
I don't think `crew remove` supports wildcards but it would be nice to run something like `crew remove llvm_*` prior to the upgrade.

Reply:
In any event, that is out of the scope of this PR.